content/post/comparing-stash-and-kubestash/index.md

```diff
@@ -0,0 +1,67 @@
+- To perform backup and restore process Stash injects a [sidecar or init-container](https://stash.run/docs/v2024.4.8/guides/workloads/overview/) to the target if it is workload (i.e. `Deployment`, `DaemonSet`, `StatefulSet` etc.). This causes the workload to restart, which is generally not preferred by users in most cases. To mitigate this issue, KubeStash introduces [Job Model](https://kubestash.com/docs/v2024.3.16/guides/workloads/overview/) to perform backup and restore process. 
+- In Stash, user has to provide the backend (where the backed up data will be stored) specification in `Repository` CR. Since, `Repository` CR has one-to-one mapping with the target application, user has to repeat the backend information in every `Repository` CR they create in the cluster. To solve this issue, we've introduced [BackupStorage](https://kubestash.com/docs/v2024.3.16/concepts/crds/backupstorage/) CR in KubeStash and moved the backend information into it. This will allow the users re-use the backend within a namespace or within the cluster.
+- In KubeStash, the Repositories and Snapshots linked to a `BackupStorage` is uploaded in the backend as metadata. When a `BackupStorage` is created with existing backup data, KubeStash automatically synchronizes the Repositories and Snapshots linked to it from the backend. There is no such feature in Stash.
+- In KubeStash, we've introduced a new CRD named [Snapshot](https://kubestash.com/docs/v2024.3.16/concepts/crds/snapshot/) which represents the state of a backup run for one or more components of an application. For every `BackupSession`, it creates `Snapshot` CRs. If a `BackupSession` involves multiple repositories, a `Snapshot` is created for each repository. This makes the task easier to restore specific snapshot and components of an application. To restore specific snapshot in Stash was more complex and tiresome process.
+- In KubeStash, we've introduced a new CRD named [RetentionPolicy](https://kubestash.com/docs/v2024.3.16/concepts/crds/retentionpolicy/) which specifies how the old Snapshots should be cleaned up. This is re-usable across namespaces. By this, users are able to specify that they want to remove backup data that are older than particular duration. In Stash, there is no option for re-usable retention policy.
+- In KubeStash, we've introduced a new CRD named [HookTemplate](https://kubestash.com/docs/v2024.3.16/concepts/crds/hooktemplate/) which basically specifies a template for an action that will be executed before or/and after backup/restore process. This is re-usable across namespaces. In Stash, there is no option for re-usable hook.
```

Comment:
In KubeStash, we've introduced a new CRD named which specifies a template for an action that will be executed before or/and after backup/restore process. It provides better control over the backup process by letting you use custom variable in the BackupBlueprint. It is also reusable across namespaces. In contrast, in Stash, you need to specify the hook separately in every BackupConfiguration/RestoreSession.